packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -1,21 +1,71 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
 import { getDisplayCode } from '../test/utils';
```

Comment:
I think you'll want to bump this line to the imports below the newline. It's a local import, whereas the two above are third-party imports.